tensorflow/compiler/mlir/tensorflow/translate/import_model.cc

```diff
@@ -2387,7 +2387,8 @@ GraphDefImporter::GetArgsRetsAndTypesFromFunctionGraph(
                                      "' is missing attribute 'index'");
 
     auto index = attr->i();
-    if (nodes->size() < index + 1) nodes->resize(index + 1);
+    const int nodes_size = nodes->size();
+    if (nodes_size < index + 1) nodes->resize(index + 1);
```

Comment:
```suggestion
    const int num_nodes = nodes->size();
    if (num_nodes < index + 1) nodes->resize(index + 1);
```

Though I think it would have been better to change type of `index`.